test/unit/sdk.test.js

```diff
@@ -1006,5 +1090,33 @@ describe('sdk', () => {
       expect(mockRemoveEventListener)
         .toHaveBeenCalledWith('message', smartcar.messageHandler);
     });
+
+    test('unmount removes the click eventListeners attached by addClickHandler', () => {
+      const id = 'connect-car-button';
+      const className = 'connect-button-class';
+
+      // mock dom & event listeners
+      document.body.innerHTML = `<div>
+      <button id="${id}">Connect your car</button>
+      <button class="${className}">Connect your car</button>
+      </div>`;
+      document.getElementById(id).addEventListener = jest.fn();
+      document.querySelectorAll(`.${className}`)
+        .forEach((element) => { element.addEventListener = jest.fn(); });
+
+      const smartcar = new Smartcar(options);
+      const clickHandlerOptions = {
+        id,
+        selector: `.${className}`,
+        state: dialogOptions.state,
+        forcePrompt: dialogOptions.forcePrompt,
+      };
+      smartcar.addClickHandler(clickHandlerOptions);
+      smartcar.unmount();
+
+      expect(document.getElementById(id).addEventListener).toHaveBeenCalledTimes(1);
```

Comment:
how do these expectations test the `unmount()`?

Reply:
Sorry forgot to change to removeEventListener after copy-and-pasting :bowing_woman: